manim/mobject/geometry.py

```diff
@@ -1554,3 +1556,243 @@ def __init__(self, main_shape, *mobjects, **kwargs):
             sub_direction = "CW"
         for mobject in mobjects:
             self.append_points(mobject.force_direction(sub_direction).get_points())
+
+
+class Angle(Arc, Elbow):
+    """A circular arc or elbow-type mobject representing an angle of two lines.
+
+    Parameters
+    ----------
+    line1 : :class:`Line`
+        The first line.
+    line2 : :class:`Line`
+        The second line.
+    radius : :class:`float`
+        The radius of the :class:`Arc`.
+    quadrant : Sequence[:class:`int`]
+        A sequence of two :class:`int` numbers determining which of the 4 quadrants should be used.
+        The first value indicates whether to anchor the arc on the first line closer to the end point (1)
+        or start point (-1), and the second value functions similarly for the end (1) or start (-1) of the second line.
+        Possibilities: (1,1), (-1,1), (1,-1), (-1,-1).
+    other_angle : :class:`bool`
+        Toggles between the two possible angles defined by two points and an arc center. If set to
+        False (default), the arc will always go counterclockwise from the point on line1 until
+        the point on line2 is reached. If set to True, the angle will go clockwise from line1 to line2.
+    dot : :class:`bool`
+        Allows for a :class:`Dot` in the arc. Mainly used as an convention to indicate a right angle.
+        The dot can be customized in the next three parameters.
+    dot_radius : :class:`float`
+        The radius of the :class:`Dot`. If not specified otherwise, this radius will be 1/10 of the arc radius.
+    dot_distance : :class:`float`
+        Relative distance from the center to the arc: 0 puts the dot in the center and 1 on the arc itself.
+    dot_color : :class:`~.Colors`
+        The color of the :class:`Dot`.
+    elbow : :class:`bool`
+        Produces an elbow-type mobject indicating a right angle, see :class:`RightAngle` for more information
+        and a shorthand.
+    kwargs
+        Further keyword arguments that are passed to the constructor of :class:`Arc` or :class:`Elbow`.
+
+    Examples
+    --------
+    The first example shows some right angles with a dot in the middle while the second example shows
```

Comment:
Isn't it a bit redundant to have an example for RightArcAngle  here and also in its own separate class? The sister class can be mentioned in a `See Also` section and the example for the RightArcAngle left only under the documentation for RightAngle.

Reply:
This example shows arc-like right angles, that is, an arc with a dot in the middle. The class `RightAngle` gives elbow-like indicators for right angles. So those examples just show that both conventions can be used in manim. Does this make sense?

Reply:
That does seem reasonable, my bad.

Reply:
All good!